Models/Settings/LogicMode.cs

```diff
@@ -4,6 +4,7 @@ public enum LogicMode
     {
         Casual,
         Glitched,
+        Accessible,
```

Comment:
As I mentioned to you, I don't want to have the discord server's custom accessible settings built into the program. You'll have to provide users with a config file that they can load up. Feel free to have the presets be able to handle a custom logic filename or the full information for logic, so that you don't need to provide a separate file for the accessible logic.